samples/apps/next-native-sample/README.md

```
@@ -0,0 +1,107 @@
+
+### 2. Import ThunderID Resources
+
+Open the `.env` file in your chosen folder and fill in your values:
+
+**`basic/thunderid.env`** — only one value needed:
+```
+SAMPLE_APP_FLOW_SECRET=sample-app-flow-secret
+```
+
+**`multi-auth/thunderid.env`** — also requires social IdP and SMS credentials:
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)